### PR TITLE
Add timestamp into the global variable containing the logger config

### PR DESCRIPTION
**Pull request by jcombes**
Usage:

``` javascript
// Setup logger
logger.setup(config.logging);
logger.splash(app, config);

// Update the logger configuration each time the file 'local.yml' is updated
fs.watchFile('config/local.yml', function(curr, prev) {
    if (curr.mtime !== prev.mtime) {
        delete require.cache[require.resolve("config")];
        var loggingCfg = require("config").logging;
        if (logger.setup(loggingCfg)) {
            logger.notice("Logger configuration has been updated...");
            logger.splash(app, loggingCfg);
        }
    }
});
```

### CODE REVIEW

**Comment by jcombes** [comment]
@sebz, can you review my modification ?

File: techLogger.js [hunk 1]
```diff
@@ -125,58 +162,75 @@ function _log(level, log) {
...
-        global.NODE_TECH_LOGGER_CFG = config;
+        _setLoggerConfig({});
+        loggerConfig = _getLoggerConfig();
 
         var msg = "\n\n!!! =========================================== !!!";
         msg += "\n!!!  No configuration set for node-tech-logger  !!!";
         msg += "\n!!! =========================================== !!!\n\n";
         console.warn(msg);
     }
+    console.info("Logger has been configured");
```
Comment (sebz) [inline techLogger.js]: At that time, the loggers have not been configured yet :stuck_out_tongue_winking_eye:
Shouldn't this be moved at the end of the function?

**Comment by sebz** [comment]
Looks good!
@jcombes can you update the `README.md` example with yours?

File: techLogger.js (hunk 1 above)
Comment (jcombes) [inline techLogger.js]: This log is useless. It was just for testing.
I remove it.

**Pull request merged**